chatterbot/corpus/corpus.py

```diff
@@ -29,7 +29,7 @@ def read_corpus(self, file_name):
         """
         import json
 
-        with open(file_name) as data_file:
+        with open(file_name, encoding='utf-8') as data_file:
```

Comment:
You probably want to use [`io.open`](https://docs.python.org/2/library/io.html#io.open) for Python 2 support.